src/screens/Manager/ReadOnlyNanoX.js

```diff
@@ -35,8 +39,9 @@ class ReadOnlyNanoX extends PureComponent<OnboardingStepProps> {
       <View style={styles.root}>
         <TrackScreen category="Manager" name="ReadOnlyNanoX" />
         <View style={styles.image}>
-          <Image source={require("../../images/nanoXHorizontalBig.png")} />
+          <DeviceNanoAction screen="empty" width={width * 0.8} />
```

Comment:
minor feedback but ideally we should explicitly set `deviceModel="nanoX"`

Reply:
After the last version from val https://github.com/LedgerHQ/ledger-live-mobile/blob/develop/src/components/DeviceNanoAction.js#L21 it's the default prop though

Reply:
i was thinking that not providing it means "i don't care what device it is, give me a generic UI", but here we probably really want a NanoX

ok for now